src/OpenTelemetry.SemanticConventions/SemanticConventionsAttributes.cs

```diff
@@ -0,0 +1,5448 @@
+
+        /// <summary>
+        /// Prefix for 'otel.library'.
+        /// </summary>
+        public static readonly string PrefixOtelLibrary = "otel.library";
+
+        /// <summary>
+        /// Prefix for 'aws.lambda'.
+        /// </summary>
+        public static readonly string PrefixAwsLambda = "aws.lambda";
+
+        /// <summary>
+        /// Prefix for 'opentracing'.
+        /// </summary>
+        public static readonly string PrefixOpentracing = "opentracing";
+
+        /// <summary>
+        /// Prefix for 'db.cosmosdb'.
+        /// </summary>
+        public static readonly string PrefixDbCosmosdb = "db.cosmosdb";
+
+        /// <summary>
+        /// Prefix for 'otel_span'.
+        /// </summary>
+        public static readonly string PrefixOtelSpan = "otel";
+
+        /// <summary>
+        /// Prefix for 'faas_span.datasource'.
+        /// </summary>
+        public static readonly string PrefixFaasSpanDatasource = "faas.document";
+
+        /// <summary>
+        /// Prefix for 'aws'.
+        /// </summary>
+        public static readonly string PrefixAws = "aws";
+
+        /// <summary>
+        /// Prefix for 'aws.s3'.
+        /// </summary>
+        public static readonly string PrefixAwsS3 = "aws.s3";
+
+        /// <summary>
+        /// Prefix for 'graphql'.
+        /// </summary>
+        public static readonly string PrefixGraphql = "graphql";
+
+        /// <summary>
+        /// Prefix for 'messaging.destination_publish'.
+        /// </summary>
+        public static readonly string PrefixMessagingDestinationPublish = "messaging.destination_publish";
+
+        /// <summary>
+        /// Prefix for 'rpc.jsonrpc'.
+        /// </summary>
+        public static readonly string PrefixRpcJsonrpc = "rpc.jsonrpc";
+
+        /// <summary>
+        /// Prefix for 'rpc.message'.
+        /// </summary>
+        public static readonly string PrefixRpcMessage = "message";
+
+        /// <summary>
+        /// Event name for 'log-feature_flag'.
+        /// </summary>
+        public static readonly string EventLogFeatureFlag = "feature_flag";
+
+        /// <summary>
+        /// Event name for 'ios.lifecycle.events'.
+        /// </summary>
+        public static readonly string EventIosLifecycleEvents = "device.app.lifecycle";
+
+        /// <summary>
+        /// Event name for 'android.lifecycle.events'.
+        /// </summary>
+        public static readonly string EventAndroidLifecycleEvents = "device.app.lifecycle";
+
+        /// <summary>
+        /// Event name for 'feature_flag'.
+        /// </summary>
+        public static readonly string EventFeatureFlag = "feature_flag";
+
+        /// <summary>
+        /// Event name for 'rpc.message'.
+        /// </summary>
+        public static readonly string EventRpcMessage = "message";
+
+        /// <summary>
+        /// Event name for 'trace-exception'.
+        /// </summary>
+        public static readonly string EventTraceException = "exception";
```

Comment:
Events will have more things to be generated, we should not keep them in semantic attributes file

Reply:
I removed them for the moment.